husky_components/datagrid/decorators/table-view.js

```diff
@@ -591,6 +566,16 @@ define(function() {
                 level: 1
             };
 
+            if (typeof this.datagrid.options.clickCallback === 'function' ||
+                typeof this.datagrid.options.actionCallback === 'function' ||
+                this.table.rows[record.id].hasChildren === true) {
+                this.sandbox.dom.addClass($row, constants.actionClass);
+            }
+            if (typeof this.datagrid.options.clickCallback !== 'function' &&
+                this.table.rows[record.id].hasChildren === false) {
```

Comment:
hasChildren is already configurable via datagrid.options.childrenPropertyName. The row objects are only internal